test/test_nn.py

```diff
@@ -17627,6 +17627,119 @@ def test_lazy_batchnorm3d_state(self):
         self._check_lazy_batchnorm_state(nn.BatchNorm3d, nn.LazyBatchNorm3d)
         self._check_lazy_batchnorm_state(nn.BatchNorm3d, nn.LazyBatchNorm3d)
 
+    def _check_lazy_instancenorm(self, cls, lazy_cls, input_shape):
```

Comment:
Are these `_check_lazy_instancenorm*` test functions needed? Looks like they're mostly copies of the `_check_lazy_batchnorm*` test functions, and it seems like those are already written to be fairly generic since they accept `cls` and `lazy_cls`.

I suggest repurposing `_check_lazy_batchnorm*` -> `_check_lazy_norm*` and calling those from the new lazy instance norm tests.

Reply:
Done for `_check_lazy_norm` and `check_lazy_norm_pickle` but this is not possible for checking the state, so I've kept separate `_check_lazy_batchnorm_state` and `_check_lazy_instancenorm_state`.